Source/Fuse.Controls.Native/NativeRenderer.uno

```diff
@@ -7,7 +7,7 @@ using OpenGL;
 namespace Fuse.Controls.Native
 {
 	extern(Android || iOS)
-	internal interface IViewHandleRenderer : IDisposable
+	public interface IViewHandleRenderer : IDisposable
```

Comment:
Why does this need to be public?

Reply:
Users cannot implement this interface if it is internal

Reply:
Are there docs that describe how to use this stuff? (I won't hold up the PR for this reason, just a question)

Reply:
No docs. This is for advanced users that can read the code.